owncloudApp/build.gradle

```diff
@@ -115,12 +115,22 @@ android {
     buildTypes {
 
         release {
+            resValue "string", "app_name_buildtype", "@string/app_name"
+            resValue "string", "dynamic_authority", "@string/authority"
+            resValue "string", "dynamic_search_suggest_authority", "@string/search_suggest_authority"
+            resValue "string", "dynamic_document_provider_authority",  "@string/document_provider_authority"
+            resValue "string", "dynamic_file_provider_authority",  "@string/file_provider_authority"
             if (System.env.OC_RELEASE_KEYSTORE) {
                 signingConfig signingConfigs.release
             }
         }
 
         debug {
+            resValue "string", "app_name_buildtype", "Debug Owncloud"
```

Comment:
Please change App Name to `Debug ownCloud` or in my opinion `ownCloud Debug` would be a better name, but maybe not completely visible.

Reply:
Exactly, as far as I remember, you see `ownclou..` In my point of view, the main info about **Debug** was hidden. An other approach could be an other icon

Reply:
First solution could be to change App Name to `Debug ownCloud` (notice which character is lower case and upper case). But it would be better to have a Debug App Icon for a better differentiation.
@michaelstingl do we have a debug icon or is it possible to get one?

Reply:
I changed it to `Debug owncloud`